website_rate_product/views/star_rate.xml

```diff
@@ -9,8 +9,8 @@
             </xpath>
         </template>
 
-        <template id="rating_product" inherit_id="website_sale.product" active="False" customize_show="True" name="Show Product Rating">
-            <xpath expr="//h1[@itemprop='name']" position="after">
+        <template id="rating_product" inherit_id="website_variants_extra.product_variants_description_3columns" active="False" customize_show="True" name="Show Product Rating">
```

Comment:
I'm talking about dependencies you removed the dependency `website_sale `and added `website_variants_extra`, this has nothing to do with the divs or whatever, besides, if I want to install the module `website_rate_product `only, I can't, you have to inherit `website_rate_product` from `website_variants_extra` and then do the correct inheritance. https://github.com/Vauxoo/addons-vauxoo/pull/708/files#diff-2aa691e2f5886e33884e188617cd0711L10